test/typescript-karma/Dashboards/DataPool/tests.js

```diff
@@ -0,0 +1,80 @@
+
+import Highcharts from '../../../../code/es-modules/masters/highcharts.src.js';
+import HighchartsComponent from '../../../../code/es-modules/Extensions/DashboardPlugins/HighchartsComponent.js';
+import Dashboards from '../../../../code/es-modules/masters/dashboards.src.js';
+import PluginHandler from '../../../../code/es-modules/Dashboards/PluginHandler.js';
+import HighchartsPlugin from '../../../../code/es-modules/Extensions/DashboardPlugins/HighchartsPlugin.js';
+import DataGridPlugin from '../../../../code/es-modules/Extensions/DashboardPlugins/DataGridPlugin.js';
+import DataGrid from '../../../../code/es-modules/masters/datagrid.src.js';
+
+
+PluginHandler.addPlugin(DataGridPlugin);
+PluginHandler.addPlugin(HighchartsPlugin);
+DataGridPlugin.custom.connectDataGrid(DataGrid.DataGrid);
+HighchartsPlugin.custom.connectHighcharts(Highcharts);
+HighchartsComponent.charter = Highcharts;
+
+const { test } = QUnit;
+
+test('Components should be initialized with a connector', function(assert) {
```

Comment:
This test fails for me without testing anything. Why was it added?

Reply:
No, I wanted to add some test, to check the ehnancement. But I guess there are plenty of the tests checking it, so i might delete it as well.